src/components/SectionContainer.js

```diff
@@ -218,31 +216,36 @@ class SectionContainer extends React.Component {
           <Col md={8} lg={9}>
             {hearing.closed ? <WrappedSection section={closureInfoSection} canComment={false}/> : null}
             <div className="section-browser">
-              <Link className="to-hearing" to={getHearingURL(hearing)}>
-                <Icon name="angle-double-left"/>&nbsp;<FormattedMessage id="hearing"/>
-              </Link>
-              <LinkWrapper
-                disabled={!sectionNav.prevPath}
-                className={`previous ${sectionNav.prevPath ? '' : 'disabled'}`}
-                to={sectionNav.prevPath || '#'}
-              >
-                <FormattedMessage id="previous"/>&nbsp;
-                <span className="type-name">
-                  {getAttr(sectionNav.prevType || section.type_name_singular, language)}
-                </span>
-              </LinkWrapper>
-              {sectionNav.currentNum}/{sectionNav.totalNum}
-              <LinkWrapper
-                disabled={!sectionNav.nextPath}
-                className={`next ${sectionNav.nextPath ? '' : 'disabled'}`}
-                to={sectionNav.nextPath || '#'}
-              >
-                <FormattedMessage id="next"/>&nbsp;
-                <span className="type-name">
-                  {getAttr(sectionNav.nextType || section.type_name_singular, language)}
-                </span>
-                &nbsp;
-              </LinkWrapper>
+              <Pager>
+                {!sectionNav.prevPath
+                  ? <Pager.Item previous href={getHearingURL(hearing)}>
```

Comment:
Instead of `href`, we must wrap all the items to the `react-router` `Link` component.

That way, the whole page is not reloaded when the link is followed; instead, react-router takes care that only the section is changed, and the header and other unchanging parts stay on the screen, which makes for a much faster and smoother experience.